mount.go

```diff
@@ -271,13 +271,23 @@ func virtioFSStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
 
 // virtioBlkStorageHandler handles the storage for blk driver.
 func virtioBlkStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
-	// Get the device node path based on the PCI address provided
-	// in Storage Source
-	devPath, err := getPCIDeviceName(s, storage.Source)
-	if err != nil {
-		return "", err
+
+	// If hot-plugged, get the device node path based on the PCI address else
+	// use the virt path provided in Storage Source
+	if strings.Contains(storage.Source, "/dev") {
+		// Make sure the virt path is valid
+		_, err := os.Stat(storage.Source)
```

Comment:
If you are matching devices, can you use `os.Stat.Mode()` and `os.ModeDevice` to check you really got one?

Reply:
done